Assets/_Project/Scripts/DiscordController.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Discord;
+public class DiscordController : MonoBehaviour
```

Comment:
Since we can't include the Discord SDK in our repo (I've removed them for you), you'll need to update the README.md with steps on how to install the SDK to get the project running